starknet_devnet/__init__.py

```diff
@@ -18,34 +18,43 @@
 
 def _patch_pedersen_hash():
     """
-    This is a monkey-patch to improve the performance of the devnet
-    We are using c++ code for calculating the pedersen hashes
-    instead of python implementation from cairo-lang package
+    Improves performance by substituting the default Python implementation of Pedersen hash
+    with swm's Python wrapper of C++ implementation.
```

Comment:
I would not use acronyms, our users are not aware of them.
```suggestion
    with Software Mansion's Python wrapper of C++ implementation.
```

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet/pull/452/commits/8e7f066d68ff6f2de641ffd0bff524a44f6aee8d